interchain-security/dungeontest-1/join-rs-dungeon-cv.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
```

Comment:
note: having these 2 files is really bad UX for almost the same setup logic. team should codense these into a source .bash file to remove duplicate pain

Reply:
These are for reference, feel free to merge them/drop Cosmovisor/drop non-Cosmovisor option. Whatever works!